tide/master/ContentLoader.h

```diff
@@ -75,6 +75,18 @@ class ContentLoader
                const QSizeF& windowSize = QSizeF( ));
 
     /**
+     * Load all the supported files from a directory.
+     *
+     * The contents are automatically arranged in a grid accross then entire
+     * DisplayGroup.
+     * @param dirName path to a directory
+     * @param gridSize size of the grid for the contents, will be automatically
+     *        determined if left empty.
+     * @return the number of contents that could be loaded
+     */
+    int loadDir( const QString& dirName, QSize gridSize = QSize{} );
```

Comment:
yes, done. Qt annoyingly uses int everywhere instead of size_t, but that can be hidden as an implementation detail of this function.